mne_bids/path.py

```diff
@@ -56,40 +56,53 @@ class BIDSPath(object):
     space : str | None
         The coordinate space for an anatomical file. Corresponds to "space".
     split : int | None
-        The split of the continuous recording file for ``.fif`` data. Corresponds to "split".
+        The split of the continuous recording file for ``.fif`` data.
+        Corresponds to "split".
     prefix : str | None
         The prefix for the filename to be created. E.g., a path to the folder
         in which you wish to create a file with this name.
     kind : str | None
-        The filename kind. This is the entity after the last ``_`` before the extension. 
-        E.g., ``'ieeg'``.
+        The filename kind. This is the entity after the
```

Comment:
strictly speaking, entities in BIDS are key value pairs. What we call "kinds" in BIDS is perhaps rather a "modality" and sometimes a suffix?

Reply:
Will adjust this in the next PR :)

Reply:
@sappelhoff See https://github.com/mne-tools/mne-bids/issues/509#issuecomment-678931896